From 7555be44007f353852b6ab7ccfc0341860b04a3c Mon Sep 17 00:00:00 2001
From: "gregkh@suse.de" <gregkh@suse.de>
Date: Mon, 19 Sep 2011 06:43:45 -0700
Subject: [PATCH 50/91] Patch "PCI: export pcie_bus_configure_settings symbol"
 has been added to the 3.0-stable tree

This is a note to let you know that I've just added the patch titled

    PCI: export pcie_bus_configure_settings symbol

to the 3.0-stable tree which can be found at:
    http://www.kernel.org/git/?p=linux/kernel/git/stable/stable-queue.git;a=summary

The filename of the patch is:
     pci-export-pcie_bus_configure_settings-symbol.patch
and it can be found in the queue-3.0 subdirectory.

If you, or anyone else, feels it should not be added to the stable tree,
please let <stable@kernel.org> know about it.

>From debc3b778508f59696ff188f0feca271dcbfa7d9 Mon Sep 17 00:00:00 2001
From: Jon Mason <mason@myri.com>
Date: Tue, 2 Aug 2011 00:01:18 -0500
Subject: PCI: export pcie_bus_configure_settings symbol

From: Jon Mason <mason@myri.com>

commit debc3b778508f59696ff188f0feca271dcbfa7d9 upstream.

pcie_bus_configure_settings needs to be exported if the PCI hotplug
driver is being compiled as a module.

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Jon Mason <mason@myri.com>
Signed-off-by: Jesse Barnes <jbarnes@virtuousgeek.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/pci/probe.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

--- a/drivers/pci/probe.c
+++ b/drivers/pci/probe.c
@@ -1474,6 +1474,7 @@ void pcie_bus_configure_settings(struct
 	pcie_bus_configure_set(bus->self, &smpss);
 	pci_walk_bus(bus, pcie_bus_configure_set, &smpss);
 }
+EXPORT_SYMBOL_GPL(pcie_bus_configure_settings);
 
 unsigned int __devinit pci_scan_child_bus(struct pci_bus *bus)
 {
